src/app/themes.tsx

```diff
@@ -124,6 +124,25 @@ const StyleManager: React.FC<Props> = ({ children, theme = lightTheme }) => {
                     },
                 },
             },
+            MuiCheckbox: {
```

Comment:
assuming these are unavoidable? maybe I should separate MUI themes into a separate file... (not on you to do!)

Reply:
Turns out these actually did nothing, they have now been removed.

(They were made in an attempt to get the checkboxes to change colour, which we couldn't figure out how to do in an MUI way)